src/transformers/models/marian/convert_marian_tatoeba_to_pytorch.py

```diff
@@ -46,194 +47,196 @@ class TatoebaConverter:
+        # not all having three-letter codes
+        resolved_src_tags, resolved_tgt_tags = self.resolve_lang_code(a3_src, a3_tgt)
+        a2_src_tags, a2_tgt_tags = [], []
+        for tag in resolved_src_tags:
+            if tag not in self.alpha3_to_alpha2:
+                a2_src_tags.append(tag)
+        for tag in resolved_tgt_tags:
+            if tag not in self.alpha3_to_alpha2:
+                a2_tgt_tags.append(tag)
 
-        s, t = ",".join(opus_src), ",".join(opus_tgt)
+        lang_tags = dedup(a2_src_tags + a2_tgt_tags)
+        src_multilingual, tgt_multilingual = (len(a2_src_tags) > 1), (len(a2_tgt_tags) > 1)
+        s, t = ",".join(a2_src_tags), ",".join(a2_tgt_tags)
 
         metadata = {
-            "hf_name": short_pair,
+            "hf_name": model_dict["_name"],
             "source_languages": s,
             "target_languages": t,
-            "opus_readme_url": readme_url,
-            "original_repo": repo_root,
+            "opus_readme_url": f"{model_dir_url}/README.md",
+            "original_repo": "Tatoeba-Challenge",
             "tags": ["translation"],
             "languages": lang_tags,
         }
         lang_tags = l2front_matter(lang_tags)
-        metadata["src_constituents"] = self.constituents[s]
-        metadata["tgt_constituents"] = self.constituents[t]
+
+        metadata["src_constituents"] = list(GROUP_MEMBERS[a3_src][1])
+        metadata["tgt_constituents"] = list(GROUP_MEMBERS[a3_tgt][1])
         metadata["src_multilingual"] = src_multilingual
         metadata["tgt_multilingual"] = tgt_multilingual
 
-        metadata.update(extra_metadata)
-        metadata.update(get_system_metadata(repo_root))
+        backtranslated_data = ""
+        if model_dict["_has_backtranslated_data"]:
+            backtranslated_data = " with backtranslations"
 
-        # combine with Tatoeba markdown
+        multilingual_data = ""
+        if "_data_per_pair" in model_dict:
+            multilingual_data = f"* data per pair in multilingual model: {model_dict['_data_per_pair']}\n"
 
-        extra_markdown = f"### {short_pair}\n\n* source group: {metadata['src_name']} \n* target group: {metadata['tgt_name']} \n*  OPUS readme: [{opus_name}]({readme_url})\n"
+        tuned = ""
+        if "_tuned" in model_dict:
+            tuned = f"* multilingual model tuned for: {model_dict['_tuned']}\n"
 
-        content = opus_readme_path.open().read()
-        content = content.split("\n# ")[-1]  # Get the lowest level 1 header in the README -- the most recent model.
-        splat = content.split("*")[2:]
+        download = f"* download original weights: {model_dir_url}/{model_dict['release']}\n"
 
-        content = "*".join(splat)
-        # BETTER FRONT MATTER LOGIC
+        langtoken = ""
+        if tgt_multilingual:
+            langtoken = f"* a sentence-initial language token is required in the form of >>id<< (id = valid, usually three-letter target language ID)\n"
```

Comment:
Nit: can we write this string on several lines to avoid going over the 119 char limit?